src/sentry/features/__init__.py

```diff
@@ -225,6 +225,7 @@
 default_manager.add("organizations:session-replay-enable-canvas", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
 default_manager.add("organizations:set-grouping-config", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
 default_manager.add("organizations:slack-overage-notifications", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
+default_manager.add("organizations:slack-block-kit", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
```

Comment:
we're using flagr to rollout?

Reply:
probably better to use the internal options thing, good callout